src/main/resources/config/application.yml

```diff
@@ -21,10 +21,16 @@ management:
     health:
         mail:
             enabled: false # When using the MailService, configure an SMTP server and set this to true
+spring.cloud.inetutils.preferredNetworks
```

Comment:
Why not use format like: 
spring
    cloud
        inetutils
            preferredNetworks

Reply:
Or better to just remove this line as it is duplicated.